.github/workflows/chain-spec-snapshot-build.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   chains-spec:
-    runs-on: ubuntu-22.04
+    runs-on: self-hosted-ubuntu
```

Comment:
As I explained several times we want this to continue to work in forks, hence we need to only use this in the `subspace` organization. And please add version to it, we don't need "ubuntu", we need a specific version, including 20.04 for some of the jobs below due to Ubuntu 22.04 using newer version of glibc and not working on 20.04 when compiled on 22.04.